recipes-ni/ni-boot-time/files/upload_results.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env python3
+"""Script for uploading the boot time measurement to influxdb"""
+
+import argparse
+import logging
+import mmap
+import re
+import sys
+import subprocess
+from datetime import datetime
+from influxdb import InfluxDBClient, exceptions
+import requests.exceptions
```

Comment:
`requests` isn't python-standard I think. Are you missing an RDEPENDS for this?

Reply:
I'll add an RDEPENDS. Fortunately we do have `python3-requests` in the feed already.
FWIW it works w/o RDEPENDS because it gets pip installed as a dependency of `influxdb`.